scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -0,0 +1,70 @@
+// CNF-1500 assembly
+[id="ztp-deploying-disconnected"]
+= Deploying distributed units at scale in a disconnected environment
+include::modules/common-attributes.adoc[]
+:context: ztp-deploying-disconnected
+
+toc::[]
+
+Telco edge computing presents extraordinary challenges with managing hundreds to tens of thousands of clusters in hundreds of thousands of locations. These challenges require fully-automated management solutions with, as closely as possible, zero human interaction.
+
+Zero touch provisioning (ZTP) addresses the problem of provisioning new edge sites from the moment there is network connectivity to the site and until the CNF workload is deployed and running on the site nodes.
+
+ZTP eliminates the need for service providers to manually configure bare-metal equipment at remote sites. Manual intervention is eliminated. Declarative frameworks of desired end states are pushed out into the environment. Template or overlay configurations install {product-title} features that are required for CNF workloads and an end-to-end functional test suite to verify CNF related features. All Kustomize configs are entirely declarative in nature.
```

Comment:
Can you make these next few lines focused on what the user is doing? You should talk more about what "you" are doing, not what "customers" are doing. There's guidance for this scattered throughout the ISG, but

> Write clearly and directly. Wherever possible, use the second person ("you").

Reply:
Didn't write this, but taking the opportunity to clean this up as suggested, thanks :)